Chess/src/chess/Main.java

```diff
@@ -75,6 +75,7 @@ public class Main extends JFrame implements MouseListener {
     private Button start, wselect, bselect, WNewPlayer, BNewPlayer;
     public static int timeRemaining = 60;
     private static HashMap<String, Integer> stateHash;
+    private static int trivialMovesCounter = 0;

```

Comment:
Minor nitpick: the functions called this trivial**Move**Counter, whereas the variable is called trivial**Moves**Counter. Please rename to trivialMoveCounter.